.travis.yml

```diff
@@ -28,7 +28,11 @@ install:
 
 before_script:
   - conda create -n test-env -c conda-forge -c defaults -c cpape --use-local elf
+  - conda activate test-env
+  - conda install -c conda-forge -c defaults -c cpape nifty
+  - conda install -c conda-forge -c defaults z5py
+  - conda install -c conda-forge -c defaults zarr
+  - pip install pyn5
```

Comment:
Would be nicer to get all this from the conda requirements.
Do you think the suggestion I made in #2 would work?

Reply:
The thing is that it should have been doing that already, but was failing to pick up z5py for some reason - see the build of current master on my travis: https://travis-ci.org/clbarnes/elf/builds/611385732#L1492

Reply:
I think z5py was never in the setup tools requirements (I just forgot to add it).